wagtailtinymce/static/wagtailtinymce/js/vendor/tinymce/.eslintrc

```diff
@@ -1,36 +0,0 @@
-{
```

Comment:
The rest are just the new version of TinyMCE. Ignore these unless you are really bored.